### PR TITLE
Develop2.x

### PR DESCRIPTION
**Pull request by todderasesareddot**

### CODE REVIEW

File: Python/kraken/ui/GraphView/kgraph_view.py [hunk 1]
```diff
@@ -187,8 +185,6 @@ def dropEvent(self, event):
             component = componentClass(parent=self.getRig())
             component.setGraphPos(Vec2(dropPosition.x(), dropPosition.y()))
             node = KNode(self, component)
-            nodeColor = component.getComponentColor()
-            node.setColor(QtGui.QColor(nodeColor[0], nodeColor[1], nodeColor[2], nodeColor[3]))
             self.addNode(node)
```
Comment (phtaylor) [inline Python/kraken/ui/GraphView/kgraph_view.py]: this change disables node coloring being driven by the component. Any reason for this? Did it cause you guys some issues?

Comment (phtaylor) [inline Python/kraken/ui/GraphView/kgraph_view.py]: sorry, I see the new code below in knode.py. Much better thanks.

**Comment by phtaylor** [comment]
I am happy to merge this now @EricTRocks, so let me know when you've had a chance to look at the changes.

**Comment by EricTRocks** [comment]
Only thing I have an issue with is the config not being cleared. I need to setup a demo script to show how this won't work if you're switching configs between builds in the same session.

**Comment by EricTRocks** [comment]
This is actually OK. I didn't realize a new method was added to the config makeCurrent().

**Pull request merged**